nexus/types/src/deployment.rs

```diff
@@ -788,6 +810,22 @@ pub enum BlueprintZoneFilter {
     ShouldDeployVpcFirewallRules,
 }
 
+/// Filters that apply to blueprint datasets.
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Hash)]
+pub enum BlueprintDatasetFilter {
```

Comment:
Out of curiosity is there a plan to use this in omdb, the way sled filters are exposed?

Reply:
I suppose we could, though we don't have an `omdb db datasets` command currently. That said, they are accessible via inventory? But I could add that in a follow-up?